derive_builder_core/src/macro_options/darling_opts.rs

```diff
@@ -241,6 +247,80 @@ pub struct Field {
     try_setter: Flag,
     #[darling(default)]
     field: FieldMeta,
+    #[darling(skip)]
+    field_attrs: Vec<Attribute>,
+    #[darling(skip)]
+    setter_attrs: Vec<Attribute>,
+}
+
+impl Field {
+    /// Remove the `builder_field_attrs(...)` packaging around attributes meant for fields
+    /// in the builder.
+    fn unnest_attrs(mut self) -> darling::Result<Self> {
+        let mut errors = vec![];
+
+        for attr in self.attrs.drain(..) {
+            let unnest = {
+                if attr.path.is_ident("builder_field_attrs") {
```

Comment:
TODO: Replace with `builder_field_attr` (applies throughout)

Reply:
Already on it, thanks.